src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -6571,15 +6634,26 @@ private static string DisposePattern(string disposeVisibility, string className,
     //   Dispose(false);
     // }}
 
-    // {CSharpFeaturesResources.This_code_added_to_correctly_implement_the_disposable_pattern}
     {implementationVisibility}Dispose()
     {{
         // {CSharpFeaturesResources.Do_not_change_this_code_Put_cleanup_code_in_Dispose_bool_disposing_above}
-        Dispose(true);
+        Dispose(true);";
+
+            if (isSealed)
```

Comment:
I do not see any value in keeping this, over the simpler approach of inlining this:

```suggestion
            if (false)
```